### ISSUE TITLE
frames reordering?

### ISSUE THREAD

**Issue by reardenlife** [nicfit/eyeD3#326]
I am having a problem with a frames in the mp3 file.
I have a SanDisk player and for some reason it truncates (to 12 symbols or so) the artist and album names in its internal library if the file was formatted by eyeD3.
I tried using different software like Star Music Tag Editor and haven't encountered such a problem.
So I started comparing files.
The most important difference seems to be the order of frames.  In case with eyeD3 the album art is inserted right at the first place.

![Screenshot]
(https://i.ibb.co/YWY4xBR/Screenshot-2019-03-04-03-06-22.png)

And it the file formatted by Star Music Tag Editor, the order is different.

Is there any way to reorder the frames in eyeD3?

**Comment by nicfit** [comment]
So the theory is that SanDisk player does not like a initial frame that is large, and therefore misses data that come later? Or that eyeD3 is truncating data?

**Comment by reardenlife** [comment]
Oops.  The reordering wasn't the problem.

The problem was with encoding.
Star Music Tag Editor converted almost everything (with an exception of TRCK field strangely) into LATIN1 encoding.
I figured it all out by using --list-raw option of a tool id3v2 (eyeD3 doesn't seem to provide any info on the encoding of the id3 frames).

So eventually I just converted everything into latin1:

```sh
eyeD3 --encoding latin1 --to-v2.4 *.mp3
```

And SanDisk displayed everything correctly.
It is definitely a bug of SandDisk firmware and frame order doesn't affect it any how.

I am closing the issue.  Sorry for the disturbance. :)
